client/src/components/pages/meditation.tsx

```diff
@@ -1,168 +1,159 @@
+    setTheme("");
+  };
 
   return (
-    <div className="meditation-page" style={backgroundStyle}>
-      <div className="meditation-content">
-        <h1 className="title" style={{ color: "black" }}>
-          {meditationSteps[step]?.title}
-        </h1>
-        <p className="content" style={{ color: "black" }}>
-          {meditationSteps[step]?.content}
-        </p>
-
-        {step === 0 && (
-          <div>
-            {meditationSteps[step].options}
-            <button
-              onClick={handleSubmitMood}
-              className="button"
-              disabled={!mood || isSubmitting}
-            >
-              {isSubmitting ? "Submitting..." : "Start Meditation"}
-            </button>
-          </div>
-        )}
-
-        {step > 0 && step < meditationSteps.length - 1 && (
-          <div className="arrow-container">
-            <button onClick={() => setStep(step - 1)} className="arrow">
-              &#8592; Previous
-            </button>
-            <button onClick={() => setStep(step + 1)} className="arrow">
-              Next &#8594;
-            </button>
-          </div>
-        )}
+    <div
+      className="meditation-container"
+      style={{
+        backgroundImage: `url(${theme && themeBackgrounds[theme]})`,
+        backgroundSize: "cover",
+        backgroundPosition: "center",
+      }}
+    >
+      {/* Hide the theme selector if a theme is selected */}
+      {!theme && (
+        <div className="theme-selector">
+          <h2 style={{ color: "white", fontSize: "2rem" }}>Choose Meditation Type</h2>
+          <select value={theme} onChange={(e) => setTheme(e.target.value as MeditationTheme)}>
+            <option style={{ color: "white", fontSize: "2rem" }} value="">--Select a Theme--</option>
+            {Object.keys(meditationThemes).map((key) => (
+              <option key={key} value={key}>
+                {key.replace(/-/g, " ")}
+              </option>
+            ))}
+          </select>
+        </div>
+      )}
 
-        {step === meditationSteps.length - 1 && (
-          <div>
-            <h2 style={{ color: "black" }}>Thank you for meditating!</h2>
-            <button
-              onClick={() => setStep(0)}
-              className="button"
-              style={{ marginTop: "20px" }}
-            >
-              Start Again
-            </button>
-          </div>
-        )}
-      </div>
+      {theme && meditationThemes[theme] && (
+        <div className="meditation-content">
+          {step < meditationThemes[theme].length ? (
+            <>
+              <h2 style={{ color: "white", fontSize: "2rem"  }}>{meditationThemes[theme][step].title}</h2>
+              <p style={{ color: "white", fontSize: "1.5rem" }}>{meditationThemes[theme][step].content}</p>
+              <div className="navigation">
+                <button onClick={() => handleNavigation("prev")}>Previous</button>
+                <button onClick={() => handleNavigation("next")}>Next</button>
```

Comment:
I added material UI as a dependency in #55, so u can use [these](https://mui.com/material-ui/react-button/) once it's merged

Reply:
sounds good